node/pkg/watchers/sui/watcher.go

```diff
@@ -222,12 +223,12 @@ func (e *Watcher) inspectBody(logger *zap.Logger, body SuiResult) error {
 
 	seq, err := strconv.ParseUint(*fields.Sequence, 10, 64)
 	if err != nil {
-		logger.Info("Sequence decode error", zap.String("Sequence", *fields.Sequence))
+		w.logger.Info("Sequence decode error", zap.String("Sequence", *fields.Sequence))
```

Comment:
Should "decode errors" be info logs, or error logs?

Reply:
Changed to Error